storm.yaml

```diff
@@ -34,7 +43,7 @@ storm.local.dir: "storm-local"
 # role must be one of the mesos-master's roles defined in the --roles flag
 #
 mesos.framework.role: "*"
-mesos.framework.checkpoint: false
+mesos.framework.checkpoint: true
```

Comment:
Cool, I'm looking forward to trying checkpointing -- I didn't even know about it until attending MesosCon.